molgenis-scripts-core/src/main/java/org/molgenis/script/core/exception/UnknownScriptTypeException.java

```diff
@@ -0,0 +1,39 @@
+package org.molgenis.script.core.exception;
+
+import static java.lang.String.format;
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Throw when an unknown ScriptType is requested.
+ */
+// S2166 'Classes named like "Exception" should extend "Exception" or a subclass' often gives false positives at dev time
+@SuppressWarnings({ "squid:MaximumInheritanceDepth", "squid:S2166" })
+public class UnknownScriptTypeException extends ScriptException
```

Comment:
Or UnknownEntityException :) (I think in the light of yesterday's discussion the UnknownEntityType)